spark-cassandra-connector/src/main/scala/org/apache/spark/sql/cassandra/package.scala

```diff
@@ -1,5 +1,113 @@
 package org.apache.spark.sql
 
+import java.util.concurrent.TimeUnit
+
+import com.datastax.spark.connector.cql.{CassandraConnector, Schema, CassandraConnectorConf}
+import com.datastax.spark.connector.rdd.ReadConf
+import com.datastax.spark.connector.writer.WriteConf
+import com.google.common.cache.{CacheLoader, CacheBuilder}
+import org.apache.spark.sql.types.StructType
+
 package object cassandra {
 
-}
+  /**
+   * Adds a method, `cassandraTable`, to SQLContext that allows reading data stored in Cassandra.
+   */
+  implicit class CSQLContext(sqlContext: SQLContext) {
+
+    /**
+     * Create a DataFrame for a given Cassandra table of a keyspace and optional cluster, user defined schema
+     * and scan type including [[PrunedFilteredScanType]], [[BaseScanType]], [[PrunedScanType]]
+     * and [[CatalystScanType]]
+     * */
+    def cassandraTable(table: String,
```

Comment:
Parameter order inconsistency - here table is the first, followed by the keyspace. In other places of the API, the keyspace is the first, followed by the table. All are strings, so this is extremely error-prone. 

What about introducing a new class `TableRef` which would hold the (cluster, keyspace, table) triple and offered easy ways of building them with fluent API and dot syntax "cluster.keyspace.table" or "keyspace.table"? This is for a separate ticket, though. Here, please only make the order consistent.